### ISSUE TITLE
bug?: GDB stub resets to off when restarting citra

### ISSUE THREAD

**Issue by parknich081** [citra-emu/citra#6503]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Canary 2306 [Linux]

### Description of Issue

if you turn on gdb stub and restart citra, it resets to off, this is really inconvenient for debugging, as i cannot setup a launch.json in vscode, because i dont think there is a command line argument to turn it on either.

### Expected Behavior

gdb stub does not reset to off. this is extremely inconvenient for debugging

### Reproduction Steps

step 1: turn on gdb stub
step 2: restart citra
step 3: use your eyes

### Log File

not really applicable here

### System Configuration

Operating System: Kubuntu 22.04
KDE Plasma Version: 5.24.7
KDE Frameworks Version: 5.92.0
Qt Version: 5.15.3
Kernel Version: 5.19.0-41-generic (64-bit)
Graphics Platform: X11
Processor: 4 × Intel® Core™ i5-6300U CPU @ 2.40GHz
Memory: 11.6 GiB of DDR4 RAM
Graphics Processor: Mesa Intel® HD Graphics 520

**Comment by SachinVin** [comment]
Works fine for me , please upload a log after step 1 and another after 3

**Comment by parknich081** [comment]
it looks like i was running the wrong thing from terminal, still dont know how to actually launch citra from terminal

**Comment by TheGag96** [comment]
No, I'm experiencing this too... starting an app by doing `citra-qt` from the command line no longer starts the GDB stub, even if you had it turned on. Opening a new issue.
